src/common-components/Logistration.jsx

```diff
@@ -33,6 +36,13 @@ const Logistration = (props) => {
     if (renameRegisterExperiment) {
       setRegisterRenameExpVariation(renameRegisterExperiment);
     }
+    if (!csrfTokenCall) {
```

Comment:
I don't think we need to manage a state for this since hook will run only once. And, CSRF service also maintains a cache for the token so if somehow a second call is initiated it will get the cached token and won't make a second call to the backend.

Reply:
Remove local state, working fine. Thanks for suggestion.